roles/matrix-redis/defaults/main.yml

```diff
@@ -0,0 +1,10 @@
+matrix_redis_enabled: true
```

Comment:
Enabling Redis is a good default for the `matrix-redis` role.

Can we adjust `group_vars/matrix_servers`, to add a new `matrix-redis` section, which would disable Redis by default. Or at least hook it up like this:

```yaml
matrix_redis_enabled: "{{ matrix_synapse_workers_enabled }}"
```